hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/workflows/HandleContext.java

```diff
@@ -528,21 +535,22 @@ <T> T dispatchChildTransaction(
      * @throws IllegalArgumentException if the transaction body did not have an id
      */
     @NonNull
-    default <T> T dispatchChildTransaction(
+    default <T> T dispatchScheduledChildTransaction(
```

Comment:
Rather than create this new method, it may be cleaner to just have Schedule Service pass in the `SCHEDULED` category.  The same lines changed for the method rename could pass the category parameter instead.

Reply:
True @jsync-swirlds , I think once everything is passing we may want to consolidate to a single `dispatch()` method that receives a `DispatchModes` argument (or the like) to discriminate the desired behavior. For now, will just try to get all needed dispatch behaviors confirmed working. 😝